Makefile

```diff
@@ -118,6 +125,16 @@ vendor:
 sanity:
 	$(MAKE) vendor && git diff --stat HEAD --ignore-submodules --exit-code
 
+sanity/src:
+	$(CONTAINER_ENGINE) run \
+		--rm \
+		-t \
+		$(CONTAINER_RUNTIME_RUN_OPTS) \
+		-w /go/src/github.com/openshift/operator-framework-olm \
+		-v $(PWD):/go/src/github.com/openshift/operator-framework-olm \
+		$(SRC_IMAGE_REPO):$(SRC_IMAGE_TAG) \
+		make sanity
```

Comment:
I wasn't able to reproduce that vendor verification check locally that we were seeing in CI, so I added a target that runs that same check in the same container environment.